app/User.php

```diff
@@ -41,4 +41,8 @@ public function company()
     {
         return $this->hasOne(Company::class, "manager_id");
     }
+    public function buyers()
```

Comment:
I think it should be requests and not a buyers function to retrieve a requests of this specific user